src/sql/azdata.proposed.d.ts

```diff
@@ -1741,9 +1741,181 @@ declare module 'azdata' {
 		ownerUri: string;
 	}
 
+	export interface SchemaCompareParams {
```

Comment:
this is leftover from before the initial schema compare PR got merged. This doesn't need to be here since it's already defined in contracts.ts

Reply:
good catch. thanks